File: Auto-generated-tasks/Difficulty-3/Task-1/Solution.py
from random import *
N = 6
k1 = 0
A = [0] * N
B = [0] * N
for k1 in range(0, N):
	A[k1] = randint(9, 916)
	B[N - 1 - k1] = A[k1]
	print("A[", k1, "]=", A[k1])
for k1 in range(0, N):
	print("B[", k1, "]=", B[k1])

File: Auto-generated-tasks/Difficulty-3/Task-1/Solution.pas
Var
	N := 6, k1: integer;
	A: Array[0..N] of integer;
	B: Array[0..N] of integer;
Begin
	for k1 := 0 to N do begin
		A[k1] := Random(907) + 9;
		B[N - 1 - k1] := A[k1];
		WriteLn("A[", k1, "]=", A[k1]);
	end;
	for k1 := 0 to N do begin
		WriteLn("B[", k1, "]=", B[k1]);
	end;
End.